website/src/components/homepage/SectionHero.js

```diff
@@ -13,11 +13,10 @@ const SectionHero = () => (
     <Grid>
       <div>
         <Heading>
-          A scalable, extensible and open-source platform to build NodeJS applications
+          Headless CMS &amp; GraphQL API for Node
         </Heading>
         <Description>
-          KeystoneJS comes with first-class GraphQL support, a highly extensible architecture, and a
-          wonderful Admin UI
+          Configure your data model in JavaScript, and KeystoneJS will generate a powerful GraphQL API and CMS.
```

Comment:
Just after this paragraph, the phrase `schema` is used instead of `data model`. Might be best to keep it consistent?
<img width="798" alt="Screen Shot 2020-06-03 at 11 30 12 am" src="https://user-images.githubusercontent.com/612020/83585610-a1198f00-a58d-11ea-868d-18e725ff45f7.png">

```suggestion
          Configure your schema in JavaScript, and KeystoneJS will generate a powerful GraphQL API and CMS.
```